torch/csrc/jit/codegen/cuda/lower_thread_predicate.cpp

```diff
@@ -263,7 +263,6 @@ void ThreadPredicateMap::print() const {
   std::cout << "--------------------------------\n";
   for (const auto& kv : thread_predicates_) {
     std::cout << "T" << kv.first->name() << " {";
-    // ir_utils::ParallelTypeBitmap
```

Comment:
This comment was indicating which part is printed (since it's a complex data structure). Can you leave it in place, with the `ir_utils::` prefix removed?

Reply:
Sorry, reverted.